buildSrc/src/main/kotlin/force-jacoco.gradle.kts

```diff
@@ -1,39 +0,0 @@
-/*
- * Copyright 2022, TeamDev. All rights reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Redistribution and use in source and/or binary forms, with or without
- * modification, must retain the above copyright notice and the following
- * disclaimer.
- *
- * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
- * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
- * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
- * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
- * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
- * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
- * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
- * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
- * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
- * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
- * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
- */
-
-// TODO:2021-07-05:dmytro.dashenkov: https://github.com/SpineEventEngine/config/issues/214.
-
-allprojects {
-    configurations.all {
-        resolutionStrategy {
-            eachDependency {
-                if (requested.group == "org.jacoco") {
-                    useVersion("0.8.7")
-                }
-            }
-        }
-    }
-}
```

Comment:
Please update the `pull` script so that it removes this file when installing the latest version.

Reply:
Please also take care for removal of `deps-between-tasks.kt` because it introduces conflicts with `Dependencies.kt` (to which it was renamed). But it takes manual operation to remove the file once compiler complains about it. And it's not usually obvious. So, let's same some time for our colleagues.

Reply:
Please also don't forget to close the referenced issue.

Reply:
Can we just delete the whole destination `buildSrc` and then copy it from scratch? Is this safe for all repos? In other words, do we have repos that have custom code in their `buildSrc`?

I'm asking because this clean-up looks a bit fragile:

```bash
echo "Updating Gradle 'buildSrc' scripts"
rm -f ../buildSrc/src/main/kotlin/force-jacoco.gradle.kts
rm -f ../buildSrc/src/main/kotlin/deps-between-tasks.kt
rm -f ../buildSrc/src/main/kotlin/io/spine/internal/gradle/Scripts.kt
cp -R buildSrc ..
```

Now `pull` script always copies "on top" that leads to accumulation of old files.

Reply:
> Is this safe for all repos? 
I don't think it's a good idea. Too aggressive. People may have custom stuff there. 

Sure, we do have Git, and our colleagues would be able to restore the deleted files. But it's too annoying.
Computers should annoy people less, not more.

Reply:
> Now pull script always copies "on top" that leads to accumulation of old files.

That's why we need to clean up things with each next version. And then remove the cleanup code, after some time, too.